### PR TITLE
fix: wrong variable definition caused by REPLACE compiler directive s…

### PR DESCRIPTION
**Pull request by ap891843**
Signed-off-by: ap891843 <aman.prashant@broadcom.com>

### CODE REVIEW

File: server/src/test/java/org/eclipse/lsp/cobol/usecases/TestReplaceCompilerDirective.java [hunk 1]
```diff
@@ -89,6 +92,29 @@
           + "           MOVE 0 to {$PAYROLL-WEEK}.\n"
           + "           MOVE 0 to {$PAYROLL-GROSS-ID}.";
 
+  private static final String TEXT5 =
```
Comment (ap891843) [inline server/src/test/java/org/eclipse/lsp/cobol/usecases/TestReplaceCompilerDirective.java:95]: Have added the same text which caused an error in the demo. I am not sure how to cover this bug in the use-case test.
Not added any assert on the returned analysis (line no -167 below) as the use-case asserts those.

Comment (temanbrcom) [reply server/src/test/java/org/eclipse/lsp/cobol/usecases/TestReplaceCompilerDirective.java:95]: If I understood correctly, the problem was with a missing location for some variable. All the locations are asserted by the use case engine, so this test is enough.

**Review by temanbrcom — APPROVED** [review]
I would recommend changing the commit message prefix to "refactor" because the code to fix has not been released yet, and this one is not a user issue. Also, the commit message body should start with a verb.

**Comment by ap891843** [comment]
> I would recommend changing the commit message prefix to "refactor" because the code to fix has not been released yet, and this one is not a user issue. Also, the commit message body should start with a verb.

Done.

**Review by grianbrcom — APPROVED** [review]
:lollipop:

**Pull request merged**